src/plugins/mf_priority.cpp

```diff
@@ -266,27 +266,68 @@ static json_t *add_held_jobs (
 }
 
 
+/*
+ * Add projects to a JSON array to be added to a bank_info JSON object.
+ */
+static json_t *add_projects (const std::pair<std::string, struct bank_info> &b)
+{
+    json_t *projects = NULL;
+
+    // add any projects to a JSON array
+    if (!(projects = json_array ()))
+        goto error;
+
+    for (auto const &j : b.second.projects) {
+    }
```

Comment:
Should this empty loop be deleted?

Reply:
yup, it should definitely be deleted 🤦 thanks for catching that, will fix